src/module/helpers/index.ts

```diff
@@ -58,7 +58,7 @@ export function sendInvite(inviteData: IInvite, invite: ICreateInvite) {
     from_name: invite.fromName,
     venture_name: invite.fromVentureName,
     invite_link: makeInviteLink(params),
-    message: `Please click the link to receive updates about ${invite.fromVentureName} on Venturemark.co`,
+    message: `Please click the link to receive updates about ${invite.fromVentureName} on Breadcrumb.co`,
```

Comment:
I'm not sure if the change for the url is correct cc @tfussell @marcusellison

Reply:
and there are others too. I'll leave them as they are until you guys confirm
![image](https://user-images.githubusercontent.com/28189578/145517199-2eb2a7a5-cee0-422a-b535-fb935280e98c.png)

Reply:
I just realized that these two issues goes together https://github.com/venturemark/webclient/issues/441

Reply:
I will discuss and let you know.

Reply:
I have access to manage DNS now. I should have it switched over by tomorrow then we can merge this.